openlibrary/plugins/openlibrary/js/carousel/Carousel.js

```diff
@@ -96,69 +96,64 @@ const Carousel = {
                     availability.status = 'private';
                 }
             }
-            let cover = {
+            const cover = {
                 type: 'id',
                 id: work.covers ? work.covers[0] : work.cover_id || work.cover_i
             };
-            let availabilityStatus = availabilityStatuses[availability.status];
-            let cls = availabilityStatus.cls;
-            let cta = availabilityStatus.cta;
-            let url = (cls == 'cta-btn--available') ?
-                (`/borrow/ia/${ocaid}`) : (cls == 'cta-btn--unavailable') ?
-                    (`/books/${openlibrary_edition}`) : work.key;
-            let isClickable = availability.status == 'error' ? 'disabled' : '';
+            const availabilityStatus = availabilityStatuses[availability.status] || availabilityStatuses.error;
+            const cls = availabilityStatus.cls;
+            const cta = availabilityStatus.cta;
+            const url = (cls == 'cta-btn--available') ? (`/borrow/ia/${ocaid}`) :
+                (cls == 'cta-btn--unavailable') ? (`/books/${openlibrary_edition}`) :
+                    work.key;
 
             if (!cover.id && ocaid) {
                 cover.type = 'ia';
                 cover.id = ocaid;
             }
 
-            return `${'<div class="book carousel__item slick-slide slick-active" ' +
-                '"aria-hidden="false" role="option">' +
-                '<div class="book-cover">' +
-                  '<a href="'}${work.key}" ${isClickable}>` +
-                    `<img class="bookcover" title="${
-                        work.title}" ` +
-                      `src="//covers.openlibrary.org/b/${cover.type}/${cover.id}-M.jpg">` +
-                  '</a>' +
-                '</div>' +
-                '<div class="book-cta">' +
-                  `<a class="btn cta-btn ${cls}" href="${url
-                  }" data-ol-link-track="subjects" ` +
-                    `title="${cta}: ${work.title
-                    }" data-key="subjects" data-ocaid="${ocaid}">${cta
-                    }</a>` +
-                '</div>' +
-              '</div>';
+            const $el = $(`
+                <div class="book carousel__item">
+                    <div class="book-cover">
+                        <a href="${work.key}">
+                            <img class="bookcover" src="//covers.openlibrary.org/b/${cover.type}/${cover.id}-M.jpg">
+                        </a>
+                    </div>
+                    <div class="book-cta">
+                        <a class="btn cta-btn ${cls}"
+                           data-ol-link-track="subjects"
+                           data-key="subjects"
+                       >${cta}</a>
+                    </div>
+                </div>`);
+            $el.find('.bookcover').attr('title', work.title);
+            $el.find('cta-btn')
```

Comment:
```suggestion
            $el.find('.cta-btn')
```

Reply:
Good eye! Haha, that's what I get for not testing actually clicking the button :D

Reply:
Fixed